### ISSUE TITLE
bug(Date Picker): Date is not selected from calendar in Android talkback on double tap 

### ISSUE THREAD

**Issue by ankushgoyal27** [angular/components#22891]
#### Reproduction

Use StackBlitz to reproduce your issue: 
- Components: https://material.angular.io/components/datepicker/overview

Steps to reproduce:
1.  Open calendar
2.  Turn on talkback
3. Try to select date and tap twice to select the date. 
 
 
#### Expected Behavior

Date should select and close the popup

#### Actual Behavior

What behavior did you actually see?
When i tap with 2 finger over the date, then date is getting selected.

Date is not selected and doesnt return to text field with date.

#### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):  Android

**Comment by zarend** [comment]
I am unable to reproduce. We have made a few bug fixes to the datepicker, and maybe this has already been fixed. @ankushgoyal27 does the issue persist?

**Comment by zarend** [comment]
Hello @ankushgoyal27 ,

Thank you for reporting. I believe we may have fixed this issue in v15. I was wondering if we could check with you if this issue requires further action. Does upgrading to v15 resolve the issue for you?

Thanks,
Zach

**Comment by zarend** [comment]
Hello @ankushgoyal27 ,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. If there are still problems with the Date Picker, please reopen this issue or file a new issue..

- Zach
